Sources/SwiftProtobuf/NameMap.swift

```diff
@@ -0,0 +1,155 @@
+// Sources/SwiftProtobuf/fieldNameMap.swift - Bidirectional field number/name mapping
```

Comment:
Update comment for new name.

Looks like git didn't think this came from the other file.  :(